tracer/test/benchmarks/Benchmarks.Trace/AppSecBodyBenchmark.cs

```diff
@@ -64,27 +70,41 @@ private void ExecuteCycle(object body)
         {
             var span = new Span(new SpanContext(1, 1), DateTimeOffset.UtcNow);
 #if !NETFRAMEWORK
-            security.CheckBody(httpContext, span, body);
-            var context = httpContext.Features.Get<IContext>();
+            _security.CheckBody(_httpContext, span, body);
+            var context = _httpContext.Features.Get<IContext>();
             context?.Dispose();
-            httpContext.Features.Set<IContext>(null);
+            _httpContext.Features.Set<IContext>(null);
 #else
-            var securityTransport = new SecurityCoordinator(security, httpContext, span);
+            var securityTransport = new SecurityCoordinator(_security, _httpContext, span);
             var result = securityTransport.RunWaf(new Dictionary<string, object> { { AddressesConstants.RequestBody, ObjectExtractor.Extract(body) } });
-            var context = httpContext.Items["waf"] as IContext;
+            var context = _httpContext.Items["waf"] as IContext;
             context?.Dispose();
-            httpContext.Items["waf"] = null;
+            _httpContext.Items["waf"] = null;
 #endif
         }
 
         [Benchmark]
-        public void AllCycleMoreComplexBody() => ExecuteCycle(complexModel);
+        public void AllCycleMoreComplexBody() => ExecuteCycle(_complexModel);
 
         [Benchmark]
         public void ObjectExtractorSimpleBody() => ObjectExtractor.Extract(new { });
 
         [Benchmark]
-        public void ObjectExtractorMoreComplexBody() => ObjectExtractor.Extract(complexModel);
+        public void ObjectExtractorMoreComplexBody() => ObjectExtractor.Extract(_complexModel);
+
+        // NOTE: these next five benchmarks are useful to help understand how the size of an
+        // object (graph) affects the ObjectExtractor, but are fail slow, so not worth running
+        // in the CI
+
+        public void ObjectExtractorProps10() => ObjectExtractor.Extract(_props10);
```

Comment:
Would be nice to see a "concurrency" comparison too, e.g. run 100/1000 extractions in parallel to check impact of lock contention vs concurrent dictionary

Reply:
done in 4f7fa5c48